frontend/src/scenes/insights/utils.ts

```diff
@@ -24,9 +24,8 @@ export const getDisplayNameFromEntityFilter = (
     filter: EntityFilter | ActionFilter | null,
     isCustom = true
 ): string | null => {
-    // Make sure custom name isn't a blank string
+    // Make sure names aren't blank strings
     const customName = (filter?.custom_name ?? '').trim() === '' ? null : filter?.custom_name
-
     const name = (filter?.name ?? '').trim() === '' ? null : filter?.name
```

Comment:
With 28 and 29 being identical _and_ needing a comment I'd be tempted to extract an `ensureStringIsNotBlank` method

Saves a future reader having to grok they're the same bar the input

Reply:
good point, I'll move this into a util